katello/katello-backup

```diff
@@ -7,14 +7,17 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
```

Comment:
I find `pulp` being a database to be quite confusing here given `mongodb` is the Pulp database. Keeping the Pulp content logically separated is a useful readability for understanding the script in my opinion.